prow/infra/service.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ingress-nginx-controller
+  namespace: ingress-nginx
+  annotations:
+    loadbalancer.openstack.org/keep-floatingip: "true"
+spec:
+  loadBalancerIP: 188.95.227.72
```

Comment:
can we set it as envvar and not put it in git?

Reply:
Why? It would require a templating step to make the manifests usable then. It is not like this IP is secret, anyone can find it with `host prow.apps.test.metal3.io`.

Reply:
I was thinking if adding a publicly reachable IP in the manifest in git increases attack surface? But yeah its also true that it is not something that is a secret after all.